content/ui-patterns/danger.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Destructive (danger) sequences
+---
+
+When users are making a destructive change to their GitHub account, we need to make sure they have a clear understanding of what is about to happen. Many times these changes are reversible, but may have irreversible side effects (making a repository public). Other times, they're not reversible at all (deleting an organization). Either way, we need to ensure that we've done everything we can to make sure there are no gaps in communication that may cause unintended effects so that we don't lose trust with users.
+
+There are a couple types of data that will need to be modified through our UI. Some of these types are not critical and require no particularly special treatment. These non-critical destructive flows include deleting a comment, issue, pull request, or discussion.
```

Comment:
Some notes here:
- Could be: "There are some types of data" (instead of "a couple")
- I'm not sure I follow the first sentence though. Would it be easier to determine that this paragraph deals with non-critical destructive flows, and the follow paragraph deals with critical destructive flows if the order of sentences were reversed: "Non-critical destructive flows include deleting a comment, issue, pull request, or discussion. These types of data need to be modified through the UI [still not sure what this means exactly], most aren't critical and require no special treatment."

Reply:
Ah - initially, that was going to lead directly into: "critical and non-critical", which is where "a couple" came from.

I'll fix that. Thanks Yaili!